omero/developers/Web/CreateApp.txt

```diff
@@ -121,6 +144,17 @@ Add the essential files to your app
             """
             return HttpResponse("Welcome to your app home-page!")
 
+-  Create :file:`<organisation_appname>/apps.py`
+
+   ::
```

Comment:
This can be moved to the end of the line of test to save all this extra whitespace